runtime/src/plugin.rs

```diff
@@ -222,6 +223,11 @@ impl Plugin {
 
         let engine = Engine::new(&config)?;
         let (manifest, modules) = manifest::load(&engine, wasm)?;
+        if modules.len() <= 1 {
+            anyhow::bail!("No wasm modules provided");
```

Comment:
Has the kernel been injected by this point? (I dimly recall that we were injecting it while loading the manifest)

Reply:
Yeah, `manifest::load` returns the user modules in addition to the kernel module

Reply:
_slaps forehead_ Aaah yeah, gotcha – I was misreading the comparison, sorry. You're already taking that extra module into account! Please disregard 😅